### ISSUE TITLE
datahandler raster catalog needs more information

### ISSUE THREAD

**Issue by demiurg** [Applied-GeoSolutions/gips#215]
1. a. add name: short but kind of says what the band is: Surface Adjusted Veg Index
    b. update description to be more verbose that describes the data: Vegitation index based on blah blah blah.

2.  Hierarchy should be in the catalogue, ensure that we have
    Data source, Asset, Product

**Comment by justinfisk** [comment]
I have addressed 2 (will be pushed with other changes) but worth noting: products can depend on more than one asset, so you will get back back a list of assets.

**Comment by ircwaves** [comment]
@demiurg -- is this still required/coveted for the site-builder?

**Comment by demiurg** [comment]
Yes, its still relevant, but I think @remeryAGS is working on this.
